tests/integration_tests/expense_reports/fixtures.py

```
@@ -0,0 +1,126 @@
+import random
+from datetime import datetime
+
+
+today_date = datetime.now().strftime('%Y-%m-%d')
+year = datetime.now().year
+month = datetime.now().month
+day = datetime.now().day
+random_int = random.randint(1, 1000)
```

Ruff (0.14.4)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)